docs/modules/models/llms/integrations/openai.ipynb

```diff
@@ -133,7 +133,16 @@
    "id": "58a9ddb1",
    "metadata": {},
    "source": [
-    "# if you are behind an explicit proxy, you can use the OPENAI_PROXY environment variable to pass through\n",
+    "# If you are behind an explicit proxy, you can use the OPENAI_PROXY environment variable to pass through"
```

Comment:
This should not be `#`, that makes it show up as a header

Reply:
Hi, thank you for review, updated